From c9c9290ff018e61d362a7d65cea6f0f6bcf2c418 Mon Sep 17 00:00:00 2001
From: Tim Blechmann <tim.blechmann@qt.io>
Date: Mon, 25 Aug 2025 22:47:34 +0800
Subject: [PATCH 096/239] cmake: use QT_NAMESPACE property

QT_NAMESPACE is a public API to access the namespace from the Qt::Core
target, replacing the private _qt_namespace property

Change-Id: I013be62bbe9068e0a9eb77ea2ffb40ac2f2f1b2e
Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>
(cherry picked from commit a79ac8cbb483297da5cc150003962610a94f076d)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/qml/Qt6QmlBuildInternals.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/qml/Qt6QmlBuildInternals.cmake b/src/qml/Qt6QmlBuildInternals.cmake
index b617fd49cf..d82dd670fc 100644
--- a/src/qml/Qt6QmlBuildInternals.cmake
+++ b/src/qml/Qt6QmlBuildInternals.cmake
@@ -302,7 +302,7 @@ function(qt_internal_add_qml_module target)
         message(FATAL_ERROR "Do not set FOLLOW_FOREIGN_VERSIONING for module ${target}. It is already set by default for internal modules.")
     endif()
 
-    get_target_property(qt_namespace ${QT_CMAKE_EXPORT_NAMESPACE}::Core _qt_namespace)
+    get_target_property(qt_namespace ${QT_CMAKE_EXPORT_NAMESPACE}::Core QT_NAMESPACE)
     if(qt_namespace)
         list(APPEND add_qml_module_args NAMESPACE ${qt_namespace})
     endif()
-- 
2.51.2

